File: cems.API/Features/LogBrowser/Dtos/LogHeaderDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using cems.API.Models;

namespace cems.API.Features.LogBrowser.Dtos
{
    public class LogHeaderDto
    {
        public int Id { get; set; }
        public Platforms Platform { get; set; }
        public DateTime Timestamp { get; set; }

        public string ExceptionMessage { get; set; }
        public  string ExceptionType { get; set; }

        public DateTime StateChangedTime { get; set; }
        public TrackedState CurrentState { get; set; }
    }
}

File: cems-SPA/src/app/dashboard/models/LogHeader.dto.ts
import { Platforms } from 'src/app/_models/Platforms.enum';
import { TrackedState } from 'src/app/_models/TrackedState.enum';

export class LogHeader {

    public id: number;
    public platform: Platforms;
    public timestamp: Date;

    public exceptionMessage: string;
    public exceptionType: string;

    public stateChangedTime: Date;
    public currentState: TrackedState;

    constructor(logHeaderObj) {
        this.id = logHeaderObj.id;
        this.platform = logHeaderObj.platform;
        this.timestamp = logHeaderObj.timestamp;
        this.exceptionMessage = logHeaderObj.exceptionMessage;
        this.exceptionType = logHeaderObj.exceptionType;
        this.stateChangedTime = logHeaderObj.stateChangedTime;
        this.currentState = logHeaderObj.currentState;
    }
}
